### PR TITLE
Adding a Data Science Free course and a React Free Courese

### PR DESCRIPTION
**Pull request by Darkrider0007**
## What does this PR do?
Add resources
### Description
- I am adding a Data Science course and a ReactJS free course 

### Why is this valuable (or not)?
- This is in Hindi.
- This is made by experts who have worked in industries for more than 10 years in different countries. 
### How do we know it's really free?
- The courses are uploaded on YouTube.

### For book lists, is it a book? For course lists, is it a course? etc.
- Yes those are course links. 

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Darkrider0007** [comment]
@greut  @radar  @Andrea  @markbates need help to fix this error

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -467,17 +468,18 @@
...
-* [React JS Tutorials for Begineers in Hindi](https://youtube.com/playlist?list=PLu71SKxNbfoDqgPchmvIsL4hTnJIrtige) - Chai aur Code
-* [React Js Tutorials in Hindi](https://youtube.com/playlist?list=PLu0W_9lII9agx66oZnT6IyhcMIbUMNMdt) - Haris Ali Khan, CodeWithHarry
-* [React Tutorial for beginners in Hindi](https://www.youtube.com/playlist?list=PL8p2I9GklV47BCAjiCtuV_liN9IwAl8pM) - Code Step By Step
-* [React Tutorial in Hindi](https://www.youtube.com/watch?v=RGKi6LSPDLU) - CodeWithHarry
-* [ReactJS Tutorial for Beginners to Advanced](https://youtube.com/playlist?list=PL_HlKez9XCSO1g7c61SyJZE4iehJDFg_w) - Technical Suneja 
-* [ReactJS Tutorial in Hindi 2020](https://www.youtube.com/playlist?list=PLwGdqUZWnOp3aROg4wypcRhZqJG3ajZWJ) - Vinod Bahadur Thapa (Thapa Technical)
-* [ReactJS Tutorials for Beginners In Hindi](https://www.youtube.com/playlist?list=PLUVqY59GNZQNTlOnGne0G7DXnmi7CeOtc) - Pradeep Maurya
+ * [Chai aur react | with projects](https://youtube.com/playlist?list=PLu71SKxNbfoDqgPchmvIsL4hTnJIrtige&si=cgNKaiSZTEnztzVX) - Chai aur Code
```
Comment (mohdahsanrazakhan) [inline courses/free-courses-hi.md:471]: Move line number `471` to `472`. To fix alphabetical ordering.

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -254,8 +254,9 @@
 
 ### Data Science
 
-*[Data Analyst Course - Beginner's to Advance (हिंदी में) - Full Playlist](https://www.youtube.com/playlist?list=PLxzTa0VPR9ryvGSuCm4RS8aeAvOLXz9XM) - IHHPET: Industries Helping Hands Dot Com
-*[Data Science Course 2023](https://www.youtube.com/playlist?list=PLfP3JxW-T70HvifebGl3d5d5jzI1Kp0i8) - Indian AI Production
+* [Data Analyst Course - Beginner's to Advance (हिंदी में) - Full Playlist](https://www.youtube.com/playlist?list=PLxzTa0VPR9ryvGSuCm4RS8aeAvOLXz9XM) - IHHPET: Industries Helping Hands Dot Com
+* [Data Analyst Course - Beginner's to Advance (हिंदी में) - Full Playlist](https://www.youtube.com/playlist?list=PLxzTa0VPR9ryvGSuCm4RS8aeAvOLXz9XM) - IHHPET: Industries Helping Hands Dot Com
```
Comment (mohdahsanrazakhan) [inline courses/free-courses-hi.md:258]: Duplicate lines

**Review by mohdahsanrazakhan — COMMENTED** [review]
Fix your code to make your PR `ready to merge`

**Pull request merged**